src/Components/Onboarding/Hooks/useOnboarding.tsx

```diff
@@ -12,11 +12,7 @@ export const useOnboarding = ({ onClose }: UseOnboarding) => {
     Dialog: () => {
       return (
         <OnboardingProvider onClose={onClose}>
-          <OnboardingModal
-            onClose={hideDialog}
-            height={["100%", "90%"]}
-            dialogProps={{ height: ["100%", "90%"], maxHeight: 800 }}
```

Comment:
Not sure why these style props weren't just on the modal itself. Some of them aren't relevant anymore though.